matter/wifi/wf200/host_if.cpp

```diff
@@ -365,7 +365,7 @@ static void sl_wfx_connect_callback(uint8_t *mac, uint32_t status)
     }
   }
 
-      if ( (status != WFM_STATUS_SUCCESS) && retryJoin < 5) {
+      if ( (status != WFM_STATUS_SUCCESS) && retryJoin < RETRY_CNT) {
```

Comment:
Great but you hadd a another RETRY COUNT define before that was also for join and of value 5.

Reused it (with a rename pattern like I suggested)

Reply:
changed